recipe/meta.yaml

```diff
@@ -33,6 +34,11 @@ test:
   imports:
     - streamz
     - streamz.core
```

Comment:
Please add `streamz.dataframe` to imports, see https://github.com/python-streamz/streamz/blob/6d5bcf4fa390617612b2638e9739161e691c1e16/setup.py#L6